who-we-are/index.md

```diff
@@ -0,0 +1,74 @@
+---
+toc: false
+layout: default
+redirect_from:
+    - team/index.md
+---
+
+# Meet us
+
+The Foundation for Public Code was founded by Ben Cerveny, Boris van Hoytema and Arnout Schuijff in 2019 to guide international collaboration among public organizations around codebase design, quality and governance.
+
+Leveraging 30 combined years of experience working with cities and technology-enabled collaboration, Boris and Ben recognized policy and whitepapers were ineffective catalysts in this space. Their landscape analysis revealed that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
+
+Based on their experience of technological innovation in a traditional industry, Arnout and Pieter van der Does (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
```

Comment:
```suggestion
Based on their experience of technological innovation in a traditional industry, Pieter van der Does and Arnout Schuijff (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
```

Reply:
@lscheske Arnout's mentioned higher up the page - I would normally only introduce someone with their full name once. Do you think people will miss the previous reference?

Reply:
This is true, but I also see @Ainali's point about the last name seemingly applying to both in this sentence. 

I have an alternative solution: legally speaking ([Articles of Association](https://about.publiccode.net/organization/articles-of-association.html), Pieter is also a co-founder of the Foundation (4th founder). So we could mention him by full name in line 10, then in this line (14) we only have to write Pieter & Arnout. What do you think?

Reply:
Ah, I hadn't spotted Jan's comment, as it was resolved.